lib/mayaUsd/commands/baseExportCommand.cpp

```diff
@@ -119,6 +119,10 @@ MSyntax MayaUSDExportCommand::createSyntax()
         kExportMaterialsFlag,
         UsdMayaJobExportArgsTokens->exportMaterials.GetText(),
         MSyntax::kBoolean);
+    syntax.addFlag(
+        kExportMaterialUnderPrimFlag,
+        UsdMayaJobExportArgsTokens->exportMaterialUnderPrim.GetText(),
+        MSyntax::kBoolean);
```

Comment:
This new flag is necessary to make merge-to-USD and duplicateto-USD work. That is because they need all generated USD data to be under the merge or copied prim. The flag ensures its material are under that prim.